app/build.gradle

```diff
@@ -5,7 +5,7 @@ android {
     buildToolsVersion '26.0.2'
     defaultConfig {
         applicationId "com.example.krtonga.simplemajifixapp"
-        minSdkVersion 14
+        minSdkVersion 15
         targetSdkVersion 25
         versionCode 1
```

Comment:
Am suggesting to look at this [article](https://medium.com/@maxirosson/versioning-android-apps-d6ec171cfd82) for better versioning. Its not mandatry